src/edu/washington/escience/myria/util/SQLiteUtils.java

```diff
@@ -119,4 +119,81 @@ public static void createTable(final String dbFileAbsolutePath, final RelationKe
       }
     }
   }
+
+  /**
+   * Deletes a relation in SQLite.
+   * 
+   * @throws SQLiteException if SQLite error occur
+   * @param dbFileAbsolutePath the SQLite file absolute path
+   * @param relationKey the relation key to delete
+   * */
+  public static void deleteTable(final String dbFileAbsolutePath, final RelationKey relationKey) throws IOException,
+      SQLiteException {
+    SQLiteConnection sqliteConnection = null;
+    SQLiteStatement statement = null;
+    try {
+      final File f = new File(dbFileAbsolutePath);
+
+      if (!f.getParentFile().exists()) {
```

Comment:
Yes, but I included this method to check if the delete call still works even though some of the workers do not have that relation. See DbDeleteTest.java (third scenario)